docs/changelog/1_0_a5.rst

```diff
@@ -0,0 +1,60 @@
+===========
+1.0 Alpha 5
+===========
+
+This changelog summarizes new features and breaking changes in
+EdgeDB 1.0 alpha 5 "Luhman 16".
+
+
+EdgeQL
+======
+
+* Implement casts between JSON and enums, arrays and tuples
+  (:eql:gh:`#251`).
+* Allow constraints on tuple types (:eql:gh:`#1576`).
+* Allow constraints directly on object types in SDL (:eql:gh:`#1164`)
+* Disallow using :eql:constraint:`exclusive` on scalar types
+  (:eql:gh:`#1575`).
+* Proper implementation of ``SET/DROP OWNED``.
+* Fix issues with some ``FOR`` statements (:eql:gh:`#1594`).
+* Use fully-qualified names to disambiguate the expressions produced
+  by :eql:stmt:`DESCRIBE` (:eql:gh:`#1254`).
+* Initial implementation of ``INSERT ... ON CONFLICT ... ELSE``
+  (:eql:gh:`#1369`)
+* Implementation of more of the features of the new migration syntax
+  (`RFC 1000 <migrations_>`_).
+
+
+GraphQL
+=======
+
+* Allow several mutation operations in a single mutation query
+  (:eql:gh:`#1569`).
+* Reflect nested aliased types (:eql:gh:`#722`).
+* Enable sorting on non-trivial path (:eql:gh:`#1642`).
+* Add an ``exists`` filter operation (:eql:gh:`#1655`).
+
+
+CLI
```

Comment:
Add "JavaScript binding" section and mention https://github.com/edgedb/edgedb-js/commit/fe5c1a605683ede999c367ff725bececfcebeccc

Reply:
With an example also

Reply:
Our bindings-related info appeared in "Misc" category last time, so I can rename that to "Bindings" and add the transactions in js there.